src/cmd-koji-upload

```diff
@@ -117,6 +122,110 @@ class Build(_Build):
     """
     Koji implementation of Build.
     """
+    def __init__(self, *args, **kwargs):
+        self._tmpdir = tempfile.mkdtemp(prefix="koji-build")
+        _Build.__init__(self, *args, **kwargs)
+
+    def __del__(self):
```

Comment:
Future: This may makes sense to be pushed up into the `_Build` class in the future.